README.md

```diff
@@ -90,3 +90,91 @@ $ docker build . -t <IMAGE_TAG>
 ### Format
 
 Formatting for `PrivateKey` or `Peer` follows the format in [Nekoyume Project README][../README.md].
+
+## How to run NineChronicles Standalone on AWS EC2 instance using Docker
+
+### A) On Your Local
+
+#### Pre-requisites
+
+- Docker environment: [Docker Installation Guide](https://docs.docker.com/get-started/#set-up-your-docker-environment)
+- Docker hub account: [Docker Hub Guide](https://docs.docker.com/docker-hub/)
+
+#### 1. Build docker image with the tag name in [<DOCKER_HUB_ACCOUNT>/<IMAGE_NAME>] format.
+
+```
+$ docker build . --tag 9c/9c-standalone --build-arg COMMIT=9c-1
+
+Usage: docker build . --tag [<DOCKER_HUB_ACCOUNT>/<IMAGE_NAME>] --build-arg COMMIT=[<VERSION_SUFFIX>]
+```
+- [Docker Build Guide](https://docs.docker.com/engine/reference/commandline/build/)
+
+![Docker Build](https://i.imgur.com/vc6CnbR.png)
+
+#### 2. Push your docker image to your docker hub account.
+
+```
+$ docker push 9c/9c-standalone:latest
+
+Usage: docker push [<DOCKER_HUB_ACCOUNT>/<IMAGE_NAME>] : [<VERSION>]
+```
+- [Docker Push Guide](https://docs.docker.com/engine/reference/commandline/push/)
+
+![Docker Push](https://i.imgur.com/IRIXXjg.png)
```

Comment:
It seems that publish images on the 'Docker Hub' goes beyond the scope of this document. Why don't we how to pull and run from our official repository?

Reply:
I wasn't aware that there was a Planetarium Docker Hub repository while I was writing this tutorial 😅 . Should I delete the build & push section and update the tutorial to pull the image from our docker hub repo? Or should I just add a section between A) and B) saying that there's a way to pull the image directly from docker hub?

Reply:
In the build section, it seems good to guide how to build a Docker image from this GitHub repository. Maybe we can get some inspiration from [this document. ](https://github.com/factoriotools/factorio-docker#quick-start)